backend/cfg/cfg_comballoc.ml

```diff
@@ -88,8 +88,8 @@ let find_compatible_allocations :
         else { allocations = List.rev allocations; next_cell = Some cell }
       | Op (Begin_region | End_region) -> (
         match curr_mode with
-        | Lambda.Alloc_local -> return ()
-        | Lambda.Alloc_heap ->
+        | Cmm.Alloc_mode.Local -> return ()
```

Comment:
I'm surprised you can't just write `Local` here - can you?

Reply:
There are various other occurrences of this elsewhere e.g. in `emit.mlp` - please omit the module path if possible.